plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMaterializedViews.java

```diff
@@ -95,6 +95,18 @@ public void testSystemMaterializedViewProperties()
                 .contains(new MaterializedRow(DEFAULT_PRECISION, "iceberg", "partitioning", "[]", "array(varchar)", "Partition transforms"));
     }
 
+    @Test
+    public void testSessionCatalogSchema()
+    {
+        Session session = Session.builder(getSession())
+                .setCatalog(null)
```

Comment:
It should be other way around.
You should provide session catalog and schema (e.g `tpch.sf1`) and have:
```
CREATE MATERIALIZED VIEW iceberg.tpch.materialized_view_session_test AS SELECT * FROM nation;
```
Then `SELECT COUNT(*) FROM iceberg.tpch.materialized_view_session_test` would fail in previous code.

Reply:
Current test also catches the bug because it hits the below check in materialised view definition constructor:
```
if (catalog.isEmpty() && schema.isPresent()) {
            throw new IllegalArgumentException("catalog must be present if schema is present");
}
```